robot_smach_states/src/robot_smach_states/rise.py

```diff
@@ -0,0 +1,67 @@
+#! /usr/bin/env python
+# ROS
+
+import smach
+
+from robot_skills.util.kdl_conversions import VectorStamped
+
+
+class HeroHmiPose(smach.State):
+    """
+    State to ensure that hero's arm is out of the way so that the robot
+    may move his torso up.
+    :param robot: Robot to execute state with
+    """
+    def __init__(self, robot):
+        smach.State.__init__(self, outcomes=['succeeded', 'failed'])
+        self._robot = robot
+
+    def execute(self, _):
+        rotation = 1.57
```

Comment:
Could it be convenient for these to be parameters that you can provide to this state? Just use these as defaults but allow for them to be provided as well?

Reply:
Made the parameters arguments for the HeroHmiPose state